Hymns/Audio Player/AudioPlayerView.swift

```diff
@@ -0,0 +1,85 @@
+import SwiftUI
+import AVFoundation
+import Combine
+
+enum PlaybackState: Int {
+    case waitingForSelection
+    case buffering
+    case playing
+}
+
+struct AudioView: View {
+
+    let player = AVPlayer()
+    let item: URL?
+    fileprivate let seekDuration: Float64 = 5
+    fileprivate var playerCurrentTime: Float64 {
+        CMTimeGetSeconds(self.player.currentTime())
+    }
+
+    @State var currentlyPlaying = true
+    @State var playbackState: PlaybackState = .waitingForSelection
+
+    var body: some View {
+        VStack {
+            AudioPlayerControlsView(player: player,
+                                    timeObserver: PlayerTimeObserver(player: player),
+                                    itemObserver: PlayerItemObserver(player: player), playbackState: $playbackState)
+            HStack(spacing: 30) {
+                //Reset button currently needed for when you tab to chords, piano, or guitar....
+                Button(action: {
+                    if self.currentlyPlaying {
+                        self.currentlyPlaying.toggle()
+                    }
+                    self.playbackState = .buffering
+                    self.player.replaceCurrentItem(with: nil)
+                    guard let url = self.item else {
+                        return
+                    }
+                    let playerItem = AVPlayerItem(url: url)
+                    self.player.replaceCurrentItem(with: playerItem)
+                    self.player.play()
+                }, label: {
+                    Image(systemName: "backward.end")
+                })
+
+                //Button to rewind 15 seconds
```

Comment:
I'm not sure where the 15 seconds comes from. `seekDuration` is only 5. Did you change it to 5 seconds instead?

(might be better to just change the comment to `// Button to rewind` and omit hte 15 seconds part)

Reply:
Addressed in commit 3678e3a